templates/workspace_services/guacamole/user_resources/guacamole-dev-vm/terraform/main.tf

```diff
@@ -60,5 +60,5 @@ output "azure_resource_id" {
 }
 
 output "connection_uri" {
-  value = "https://${data.azurerm_app_service.guacamole.default_site_hostname}/?/client/${textencodebase64("${azurerm_network_interface.internal.private_ip_address}\u0000c\u0000azuretre", "UTF-8")}"
+  value = "https://${data.azurerm_app_service.guacamole.default_site_hostname}/?/client/${textencodebase64("${azurerm_linux_virtual_machine.linuxvm.name}\u0000c\u0000azuretre", "UTF-8")}"
```

Comment:
Does this change means, that I can use EITHER the old connection uri and the new one?

I mean, with the ip or the vm name?
Or, to rephrase the question, will the change still be correct if you reverted the change in this file?

Reply:
No, as the ID of the connection is now the hostname to prevent duplicates. So not "backwards compatible". Is a breaking change. Thoughts?

Reply:
I see you added the displayname property. so I'm not 100% sure would the ip  stop working...

Reply:
I added the dispay_name property to the configuration.

The connection however has a single identifier that I changed to be hostname.